onnx/test/reference_evaluator_test.py

```diff
@@ -2914,6 +2914,61 @@ def test_cast_float_to_string_and_back(self):
         got = ref.run(None, {"X": data})[0]
         assert_allclose(got, np.array([1.152512, -0.152612, 0.0, np.nan]))
 
+    def test_split_to_sequence(self):
+        X = make_tensor_value_info("X", TensorProto.FLOAT, [None])
```

Comment:
I think you mean `None` instead of `[None]`. `None` means a tensor of unknown rank, while `[None]` means a 1-dimensional tensor of unknown dimension.